### ISSUE TITLE
How do you run this?

### ISSUE THREAD

**Issue by GenieRoach** [lilmond/Bloxburg-Autofarm#1]
Not sure exactly how to run this thing.

**Comment by lilmond** [comment]
Hi, thank you for asking.

To make this thing work, you must of course play Bloxburg. And here's the step-by-step tutorial:

Go to the hairdresser workplace. The script works only for this job, the script was originally written just for it.
Execute the script (bloxburg_autofarm_v1.lua).
Let a customer sit on your chair. It's fine if a customer is already sitting on it before you execute the script.
Click the "Done" button on the workplace mirror, it's okay if you don't submit the correct hair style for the customer, the script will only hook the RemoteEvent object for submitting it and then save it to the game's global variable aka: "getgenv()" for later use such as re-executing the script after pausing it.
You are then free to go AFK.

To pause the auto-farm, if you, by any chance had to do it, you can execute the script below.
```lua
getgenv().autofarm = false
```

If you have any more question, feel free to comment. Thanks.

**Comment by GenieRoach** [comment]
Thanks for the detailed response. I guess I should have been more specific. The above information will help... once I know how to actually execute the script. I am not familiar with Lua and how to execute the script.

**Comment by GenieRoach** [comment]
After some reading, watching, checking. I believe I am in visual code trying to execute the script wtih an error popping up on line 50, the word 'continue' is red. 
Syntax error near 'end'

**Comment by lilmond** [comment]
I see, may I know what executor do you use?

**Comment by GenieRoach** [comment]
ok so I tried to use visual code with lua 53. Followed some video's to get me there. Was I on the right path at least?
Thanks for coming back to me on this btw!

**Comment by GenieRoach** [comment]
any ideas on this mate?

**Comment by lilmond** [comment]
Hey, sorry for taking so long to respond. But do you play Roblox by any chance? It's actually a Lua script for Roblox. You can execute it using Roblox exploits.

**Comment by GenieRoach** [comment]
Yes, it is in Roblox, Welcome to Bloxburg I am trying to run this. I will look up how to run that using the Roblox exploit. Thanks

**Comment by magomarov2105-crypto** [comment]
__
